consensus/qbft/src/main/java/org/hyperledger/besu/consensus/qbft/pki/DefaultKeyStoreWrapperProvider.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See
+ *  the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.hyperledger.besu.consensus.qbft.pki;
+
+import org.hyperledger.besu.pki.keystore.HardwareKeyStoreWrapper;
+import org.hyperledger.besu.pki.keystore.KeyStoreWrapper;
+import org.hyperledger.besu.pki.keystore.SoftwareKeyStoreWrapper;
+
+import java.nio.file.Path;
+
+import com.google.common.annotations.VisibleForTesting;
+
+public class DefaultKeyStoreWrapperProvider implements KeyStoreWrapperProvider {
```

Comment:
Does this need to be a decorator? We know up front whether we want a hardware or software keystore I think.